sqlmesh/core/config/loader.py

```diff
@@ -184,8 +184,14 @@ def load_config_from_python_module(
     module_path: Path,
     config_name: str = "config",
 ) -> C:
-    with sys_path(module_path.parent):
-        config_module = import_python_file(module_path, module_path.parent)
+    try:
+        with sys_path(module_path.parent):
+            config_module = import_python_file(module_path, module_path.parent)
+    except Exception as e:
+        raise ConfigError(
+            f"Failed to load config file: {str(e)}",
```

Comment:
```suggestion
            f"Failed to load config file: {e}",
```
there is no need of str since we use post 3.6 python